sklearn/svm/tests/test_svm.py

```diff
@@ -361,12 +361,13 @@ def test_decision_function():
     assert_array_almost_equal(dec.ravel(), clf.decision_function(X))
 
 
-def test_decision_function_shape():
-    # check that decision_function_shape='ovr' gives
+@pytest.mark.parametrize('SVM', (svm.SVC, svm.NuSVC))
+def test_decision_function_shape(SVM):
+    # check that decision_function_shape='ovr' or 'ovo' gives
     # correct shape and is consistent with predict
 
-    clf = svm.SVC(kernel='linear', C=0.1,
```

Comment:
why changing `C`?

Reply:
I removed it because NuSVC doesn't support C and I wanted to parametrize the test with both classes. Also it's not relevant for checking the shapes

Reply:
fair, and the runtimes don't seem to be that different (C=1.0 may even be faster)